nilearn/image/image.py

```diff
@@ -782,7 +780,8 @@ def math_img(formula, **imgs):
 
 
 def clean_img(imgs, sessions=None, detrend=True, standardize=True,
-              confounds=None, low_pass=None, high_pass=None, t_r=2.5):
+              confounds=None, low_pass=None, high_pass=None, t_r=2.5,
+              ensure_finite=False):
```

Comment:
I think agreement was not fill zeros automatically by default. Full discussion is in #282